### ISSUE TITLE
Only one image uploading when multiple files selected

### ISSUE THREAD

**Issue by anchovy** [verbb/formie#187]
**Description**
I've got an upload field, handle = images, set to allow images only, max 6 files - but only 1 image is being uploaded - the 2nd selected.
Looking at the data sent there's only one key used of all file data `fields[images]`, would expect this, if multiple files are allowed to be `fields[images][0],fields[images][1]` etc.

![image](https://user-images.githubusercontent.com/599005/96526795-1a1a4880-12db-11eb-89fd-df2af49a8292.png)

**Steps to reproduce**
1.
2.

### Form settings
Please pick from the options below.

- Multi-page form:  No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
Plugin version: 1.2.22
Craft version: Craft Solo 3.5.8
Multi-site: No
**Additional context**

**Comment by engram-design** [comment]
You're totally correct, sorry about that. Just fixed for the next release.

To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.23.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by anchovy** [comment]
Sweet - thanks.

**Comment by engram-design** [comment]
Fixed in [1.2.24](https://github.com/verbb/formie/releases/tag/1.2.24)
